pkg/scheduler/api/types.go

```diff
@@ -157,6 +157,9 @@ type ExtenderConfig struct {
 	// so the scheduler should only send minimal information about the eligible nodes
 	// assuming that the extender already cached full details of all nodes in the cluster
 	NodeCacheCapable bool
+	// Only pods matching this selector will be sent to the extender for filter/prioritize/bind operations.
+	// +optional
+	Selector *metav1.LabelSelector
```

Comment:
Maybe mention the default value, which is an empty selector that matches every pod.